pytest_tests/testsuites/object/test_object_api.py

```diff
@@ -337,6 +341,69 @@ def test_object_search_should_return_tombstone_items(
                     object_type == "TOMBSTONE"
                 ), f"Object wasn't deleted properly. Found object {tombstone_oid} with type {object_type}"
 
+    @allure.title("Validate objects search by common prefix")
+    def test_search_object_api_common_prefix(self, default_wallet: str, simple_object_size: int):
+        FILEPATH_ATTR_NAME = "FilePath"
+        cli = NeofsCli(self.shell, NEOFS_CLI_EXEC, WALLET_CONFIG)
+        wallet = default_wallet
+        cid = create_container(wallet, shell=self.shell, endpoint=self.cluster.default_rpc_endpoint)
+
+        objects = {}
+        for _ in range(5):
+            file_path = generate_file(simple_object_size)
+
+            with allure.step("Put objects"):
+                objects[file_path] = put_object_to_random_node(
```

Comment:
The put_object_to_random_node function returns the ID.
Why do you store the ID in objects[file_path]?

Reply:
Further in test I'm using this mapping to get an ID by file_path to verify an output from the search method